### PR TITLE
Rpm  reset appx package

### PR DESCRIPTION
**Pull request by RDMacLachlan**

### CODE REVIEW

**Comment by e0i** [comment]
@dariomws @get-itips @tseward

Peer review needed for this PR.

cc @JohanFreelancer9 for copy editing.

Thank you.

**Review by get-itips — APPROVED** [review]
I think this needs to be added here also https://github.com/MicrosoftDocs/windows-powershell-docs/blob/master/docset/windows/appx/Appx.md

**Comment by get-itips** [comment]
Also the build is currently failing for this file, please check the format of the markdown with PlatyPS.

**Review by dariomws — CHANGES_REQUESTED** [review]
Agree with @get-itips.

Error is "Expect Heading, please try to use PlatyPS validate the content".

File: docset/windows/appx/Reset-AppxPackage.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: True
+Accept wildcard characters: False
+```
+
+### CommonParameters
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see about_CommonParameters (http://go.microsoft.com/fwlink/?LinkID=113216).
```
Comment (JohanFreelancer9) [inline docset/windows/appx/Reset-AppxPackage.md:69]:
```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see about_CommonParameters (https://go.microsoft.com/fwlink/?LinkID=113216).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - please see @get-itips' comments.

File: docset/windows/appx/Reset-AppxPackage.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+ms.technology: powershell-windows
+ms.topic: reference
+online version:
+schema: 2.0.0
+title: Reset-AppxPackage
+---
+
+# Reset-AppxPackage
+
+Starting at Windows 10 Insider Preview Build 20215 you get access to the `Rest-AppxPackage` PowerShell Cmdlet for use in resetting your installed Windows Apps.
```
Comment (JohanFreelancer9) [inline docset/windows/appx/Reset-AppxPackage.md:24]:
```suggestion
Starting at Windows 10 Insider Preview Build 20215 you get access to the `Rest-AppxPackage` PowerShell Cmdlet for use in resetting your installed Windows apps.
```

**Comment by e0i** [comment]
Re-opening to regenerate build.

**Comment by RDMacLachlan** [comment]
Resolved error in the article.

**Comment by e0i** [comment]
Closing and re-opening to refresh validation checks.

**Review by tseward — APPROVED** [review]

**Comment by RDMacLachlan** [comment]
Following the reference guide I have updated the ms.technology from "powershell-windows" to "windows". Should this change be made to all current articles in this repository?

**Comment by dariomws** [comment]
@RDMacLachlan thank you for your contribution.

Yes, I think it must be changed in all current articles in this repository, but give me some time to discuss it with mi colleagues.

File: docset/windows/appx/Reset-AppxPackage.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+online version:
+schema: 2.0.0
+title: Reset-AppxPackage
+---
+
+# Reset-AppxPackage
+
+## SYNOPSIS
+
+Starting at Windows 10 Insider Preview Build 20215 you get access to the `Rest-AppxPackage` PowerShell Cmdlet for use in resetting your installed Windows Apps.
```
Comment (get-itips) [inline docset/windows/appx/Reset-AppxPackage.md:26]:
```suggestion
Starting at Windows 10 Insider Preview Build 20215 you get access to the `Reset-AppxPackage` PowerShell Cmdlet for use in resetting your installed Windows Apps.
```

**Review by tseward — APPROVED** [review]
Approved pending @get-itips suggestion.

**Review by dariomws — APPROVED** [review]

**Pull request merged**